images/access-setup/content/bin/setup_compute.sh

```diff
@@ -176,6 +178,28 @@ tekton_chains_manifest(){
   printf "OK\n"
 }
 
+tekton_results_manifest(){
+  manifest="$manifests_dir/compute/tekton-results/tekton-results-secret.yaml"
+  if [ ! -e "$manifest" ]; then
+    manifests_tmp_dir="$(dirname "$manifest")/tmp"
```

Comment:
This var seems to be unused and could be removed (L184,L185,L194).

Reply:
var removed